package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "tier-mark",
+  "version": "1.0.0",
+  "description": "Tier module for FEC",
+  "main": "index.js",
+  "scripts": {
+    "start": "",
+    "test": "",
+    "seed": "node database/index.js",
+    "server-dev": "nodemon server/index.js",
+    "react-dev": "webpack -d --watch"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/theofficetroll/Tier-Mark.git"
+  },
+  "author": "Mark Scrudder",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/theofficetroll/Tier-Mark/issues"
+  },
+  "homepage": "https://github.com/theofficetroll/Tier-Mark#readme",
+  "dependencies": {
+    "babel": "^6.23.0",
```

Comment:
Do you need babel in dependencies or devDependences?  Reply to this comment explaining which and justify your choice.

Reply:
Babel should be in devdependencies, not dependencies, as it will not be needed in production. Babel and Webpack are used to transpile the code and deployed app will use bundle.js so these will not be necessary.

I have moved this for the next PR and will keep an eye on this in the future, thank you!